software/firmware/bootloader.py

```diff
@@ -89,7 +89,6 @@ def show_error(cls, title, message, duration=1):
         """
         oled.fill(0)
```

Comment:
[required] This clear is also unnecessary now, right?

Reply:
That was a leftover from an earlier revision before adding the `clear_first` argument to the `centre_text` method.

Fixed now.